MapleServer2/Types/Player.cs

```diff
@@ -129,8 +157,31 @@ public static Player MaleDefault(long accountId, long characterId, string name =
             stats.MoveSpd = new PlayerStat(110, 100, 150);
             stats.JumpHeight = new PlayerStat(110, 100, 130);
 
-            return new Player
-            {
+            List<SkillTab> skillTabs = new List<SkillTab>();
+            skillTabs.Add(new SkillTab("test1"));
+
+            // Add archer skills set to level 1 and unlearned (normally would be loaded from database)
+            XmlDocument xmlDoc= new XmlDocument();
+            xmlDoc.Load("./MapleServer2/Constants/Skills/skills_50.xml");
```

Comment:
This code looks duplicated below, maybe put it into one method that loads skills based on job (in this case 50)

Reply:
Those are hard-coded players only there for testing, so I don't think it's necessary here.